File: ticketProject4-DistributedJava/TicketSystem/TicketNetworking/src/main/java/network/objectprotocol/GetAllARFromDateResponse.java
package network.objectprotocol;

import network.dto.ArtisticRepresentationDTO;

public class GetAllARFromDateResponse implements Response {
    private Iterable<ArtisticRepresentationDTO> ARs;

    public GetAllARFromDateResponse(Iterable<ArtisticRepresentationDTO> ARs) {
        this.ARs = ARs;
    }

    public Iterable<ArtisticRepresentationDTO> getARs() { return ARs; }


}

File: ticketProject7-CrossPlatform/TS-crossp/networking/response/GetAllARFromDateResponse.cs
using System;
using System.Collections.Generic;

namespace networking.protocol
{

	using ArtisticRepresentationDTO = networking.dto.ArtisticRepresentationDTO;

	[Serializable]
	public class GetAllARFromDateResponse : Response
	{
		private IEnumerable<ArtisticRepresentationDTO> arsDTO;

		public GetAllARFromDateResponse(IEnumerable<ArtisticRepresentationDTO> arsDTO)
		{
			this.arsDTO = arsDTO;
		}

		public virtual IEnumerable<ArtisticRepresentationDTO> getArs
		{
			get
			{
				return arsDTO;
			}
		}
	}

}